spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinProperties.java

```diff
@@ -22,13 +22,17 @@
  * Zipkin settings
  *
  * @author Spencer Gibb
+ * @author Brock Overcash
  * @since 1.0.0
  */
 @ConfigurationProperties("spring.zipkin")
 public class ZipkinProperties {
 	/** URL of the zipkin query server instance. */
 	private String baseUrl = "http://localhost:9411/";
 	private boolean enabled = true;
+	private boolean basicAuthenticated = false;
+	private String username = "admin";
```

Comment:
Which lines specifically? I think basicAuthenticated should default to false, but I agree username and password don't need defaults. I included them because all of the others had defaults.